tests/unit/django_spanner/test_expressions.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2020 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+from django.test import SimpleTestCase
+from django_spanner.compiler import SQLCompiler
+from django.db.models import F
+from .models import Report
+
+
+class TestUtils(SimpleTestCase):
+    settings_dict = {"dummy_param": "dummy"}
+
+    def _get_target_class(self):
+        from django_spanner.base import DatabaseWrapper
+
+        return DatabaseWrapper
+
+    def _make_one(self, *args, **kwargs):
+        return self._get_target_class()(*args, **kwargs)
```

Comment:
Since this is repeated in every test class I'd recommend moving it into a spanner-specific base test class.

Reply:
Done.

Reply:
Done.